### DISCUSSION TITLE
Help installing pyphi

### DISCUSSION THREAD

**Discussion by manutremo** [salvadorgarciamunoz/pyphi#20] [Q&A] [answered]
Hi, I'm trying to install pyphi using the current master branch. I follow the instructions until the end (pip install -r requirements.txt), and then try to run the example file. However it seems pyphi is not found since it's not installed.

Looking at previous versions it looks like there used to be a setup.py file that took care of installation, but this file is gone in the current master branch.

I then imagined I could just download the previous version, which includes setup.py, uncompress it, and then download the current master branch and uncompress it on the same folder, which leaves me with the latest versions of all lib .py files, additional to the old setup.py.

After doing that, the example test seems to work correctly - however, I'm not sure if this is producing a fully working isntall or if this workaround will leave me to issues down the road.

Can anyone let me know if this workflow looks right at least while the master branch is updated, or if I should be doing something differently?

Many thanks in advance.

**Comment by aegeorgopoulos** [comment]
I also have been having this issue, and was able to rectify it via re-adding the setup.py file. This has not caused any known errors. Is there some way to install everything via anaconda (pyphi and its dependencies) so that no conflicts are created later?

**Comment by salvadorgarciamunoz** [comment, marked as answer]
The easiest way is to install anaconda, download the code and uncompress in a directory and then add the PYTHONPATH in Spyder. I also test via pip install -r requirements.txt and it worked ok without the need of a setup.py

**Comment by Raul-Barajas** [comment]
Hi Salvador,

I wanted to let you know that I encountered a couple of minor issues while trying to install the package from source using pip install . 

    In the setup.py file, the install_requires list includes a typo:

        "xlrd"" (line 32) contains a double closing quote, which causes a syntax error during installation.

        There is also a non-existent package listed: "datetimematplotlib". I assume this was meant to be either "datetime", "matplotlib".

I corrected these lines locally by updating the block to:

install_requires=[
    "bokeh",
    "datetime",
    "matplotlib",
    "numpy",
    "openpyxl",
    "pandas",
    "pyomo",
    "scipy",
    "statsmodels",
    "spyder",
    "xlrd"
]

After that, the installation completed successfully.

Just wanted to bring this to your attention in case you’d like to update the repository to prevent future issues for other users.

Thanks again for the tool — looking forward to working with it.

Best regards,
Raúl Barajas

**Reply by salvadorgarciamunoz** [reply]
Hi Raul,
Thanks for pointing these out, there were indeed two typos in the file, they have been corrected now.
Best
Sal
